src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx

```diff
@@ -59,12 +57,20 @@ New Relic Lookout helps software teams of any size discover potential issues bef
 * Analyze any data in New Relic database (NRDB), including third-party, open, and custom data.
 * Launch into other areas of the New Relic platform for deeper understanding
 
-## Requirements [#reqs]
+## The two Lookout experiences [#reqs]
+
+There are two versions of New Relic Lookout: 
+
+* The global version of Lookout: This version of Lookout shows you data and deviations from across all the monitored entities in your estate. This also gives you data spanning 13-months, correlations, profiles, traces, and other details. You access this by going to **All Capabilities** and clicking on **Lookout**, and it shows up on your left-side nav. 
+  * Requires being a full platform user.
+* Lookout for specific entity types: A more basic version of Lookout that applies within a specific type of monitored entity. For example, you can go to **APM and services** and access Lookout, and see data and deviations for those entity types.  
+  * Available to all user types. 
+
+## Recent changes [#updates]
 
-* New Relic Lookout requires [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions).
-* If you are on Standard edition, you can still use the New Relic Lookout view in the [New Relic Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/entity-explorer-view-performance-across-apps-services-hosts). If you want to see data spanning 13-months, correlations, profiles, traces, and other details, you must upgrade to a higher edition.
+On April 24, 2023, we made some changes to New Relic Lookout. For example, we've changed what the size of circles means. For a summary of changes, see our [What's new? post](/whats-new/2023/04/whats-new-04-20-lookout-visualization-update). 
 
-## Get started with New Relic Lookout
+## Get started with New Relic Lookout [#get-started]
 
 The default view provides insight into three key service performance indicators broken down by application: **throughput**, **response time**, and **errors**. These metrics are analyzed to show how the data has behaved during the last five minutes compared to the prior hour.
```

Comment:
Can we make this sentence more active? 'are analyzed' is a little ambiguous and I'm not sure if it's providing much context to the sentence. You might get away with 'These metrics show how the data behaved during the last five minutes compared to the previous hour.'